Doc/library/sqlite3.rst

```diff
@@ -2345,7 +2346,7 @@ can be found in the `SQLite URI documentation`_.
 How to create and use row factories
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, :mod:`!sqlite3` represent fetched rows as :class:`tuples <tuple>`.
+By default, :mod:`!sqlite3` represents each fetched row as a :class:`tuple`.
```

Comment:
@merwok , do you like it better without "fetched"?

```suggestion
By default, :mod:`!sqlite3` represents each row as a :class:`tuple`.
```

Reply:
I think we should consider leaving it in place. It is slightly more explicit with "fetched" than without. An alternative might have been "resulting rows", but "fetched rows" is better, IMO.

Reply:
Yeah; just to be clear, I don't have a particular preference—I was just presenting this as an option as @merwok had previously mentioned he wasn't a huge fan of the current construction here.

FWIW, "resulting rows" wouldn't really address that, if I'm understanding his concern correctly, and to me would be unclear and out of place, since it is never stated or even directly implied what specific action these rows are "resulting" _from_ (whereas fetched is much less semantically dependent in this fashion, and readers can reasonably infer it they are fetched from the DB or some variation thereof).